pkg/proxy/winkernel/proxier.go

```diff
@@ -235,13 +235,13 @@ func kernelSupportsDualstack(currentVersion hcn.Version) bool {
 }
 
 func Log(v interface{}, message string, level klog.Level) {
-	klog.V(level).InfoS("%s", message, "spewConfig", spewSdump(v))
+	klog.V(level).InfoS("msg", message, "spewConfig", spewSdump(v))
```

Comment:
the format of InfoS is `klog.InfoS("message", "key1", value1, "key2", "value2")`